app/controllers/UserProfileController.scala

```diff
@@ -27,6 +28,19 @@ import scala.concurrent.Future
 class UserProfileController @Inject() (implicit val env: Environment[User, SessionAuthenticator])
   extends Silhouette[User, SessionAuthenticator] with ProvidesHeader  {
 
+  // List of user IDs for researchers that have worked on Project Sidewalk. Used to filter or mark as a researcher.
```

Comment:
Repeated list from the AdminController. I think you should keep this list in a single file and refer that file in both of them. Easy to maintain.